src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(param) {
+      this.result += param;
```

Comment:
```suggestion
      calc.result += param;
```
Ты можешь использовать название переменной calc